### DISCUSSION TITLE
Player Identity > Representation > Dispersed

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#76] [Framework Discussions]
# Player Identity > Representation > Dispersed

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
There is a type of ship called an Aircraft Carrier where you're able to control both the ship's movements as well as the flight squadrons (one at a time) having, in a sense, control of two entities at the same time
